### PR TITLE
bpo-45017: move opcode-related logic from modulefinder to dis

### PR DESCRIPTION
**Pull request by iritkatriel**
https://bugs.python.org/issue45017

### CODE REVIEW

File: Lib/dis.py [hunk 1]
```diff
@@ -535,6 +535,41 @@ def findlinestarts(code):
...
+            level = consts[opargs[i-2][1]]
+            fromlist = consts[opargs[i-1][1]]
+            yield (names[oparg], level, fromlist)
+
+def find_store_names(co):
+    """Find names of variables which are written in the code
+
+    Generate sequence of strings
+    """
+    STORE_OPS = (
```
Comment (1st1) [inline Lib/dis.py]: nit: 
```suggestion
    STORE_OPS = {
```

File: Lib/test/test_dis.py [hunk 2]
```diff
@@ -1326,5 +1326,36 @@ def test_assert_not_in_with_arg_in_bytecode(self):
...
             self.assertNotInBytecode(code, "LOAD_CONST", 1)
 
+class TestFinderMethods(unittest.TestCase):
+    def test_find_imports(self):
+        cases = [
+            ("import a.b.c", ('a.b.c', 0, None)),
+            ("from a.b import c", ('a.b', 0, ('c',))),
+            ("from a.b import c as d", ('a.b', 0, ('c',))),
+            ("from ...a.b import c as d", ('a.b', 3, ('c',))),
+            ("from ..a.b import c as d, e as f", ('a.b', 2, ('c', 'e')))
```
Comment (1st1) [inline Lib/test/test_dis.py]: Add a test for star imports?

**Review by 1st1 — APPROVED** [review]
I'm not the code owner here but the refactoring looks ok to me.

File: Lib/test/test_dis.py (hunk 2 above)
Comment (iritkatriel) [reply Lib/test/test_dis.py]: I knew I forgot something.

**Pull request merged**